components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/correlation/CorrelationConfigManager.java

```diff
@@ -0,0 +1,205 @@
+/*
+ *
```

Comment:
Unwanted empty lines

Reply:
> Unwanted empty lines

These changes were authored and committed by @pasindu-sandima. I will send a separate PR adding these requested changes since this PR is already merged.